server/routes/activities/waitlist-application/handlers/activity.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from 'express'
+import { Expose, Type } from 'class-transformer'
+import { Min } from 'class-validator'
+import ActivitiesService from '../../../../services/activitiesService'
+
+export class Activity {
+  @Expose()
+  @Type(() => Number)
+  @Min(1, { message: 'Select an activity' })
+  activityId: number
+}
+
+export default class ActivityRoutes {
+  constructor(private readonly activitiesService: ActivitiesService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+
+    const activities = await this.activitiesService
+      .getActivities(true, user)
+      .then(act => act.filter(a => a.category.code !== 'SAA_NOT_IN_WORK'))
+      .then(act => act.map(a => ({ id: a.id, name: a.description })))
+
+    return res.render(`pages/activities/waitlist-application/activity`, { activities })
+  }
+
+  POST = async (req: Request, res: Response): Promise<void> => {
+    const { activityId } = req.body
+    const { prisoner } = req.session.waitListApplicationJourney
+    const { user } = res.locals
+
+    const activity = await this.activitiesService.getActivity(activityId, user)
+
+    const alreadyAllocated = await this.activitiesService
+      .getActivePrisonPrisonerAllocations([prisoner.prisonerNumber], user)
```

Comment:
I presume the intention here is to extend this to check waiting lists as well?  This will only check allocations.  No endpoint to check this at the mo.